runtime/cere/src/lib.rs

```diff
@@ -125,7 +125,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// and set impl_version to 0. If only runtime
 	// implementation changes and behavior does not, then leave spec_version as
 	// is and increment impl_version.
-	spec_version: 48900,
+	spec_version: 48902,
 	impl_version: 0,
 	apis: RUNTIME_API_VERSIONS,
 	transaction_version: 14,
```

Comment:
```suggestion
	transaction_version: 15,
```
Since we've added a new extrinsic call, `transaction_version` should be incremented.

Reply:
Updated